### ISSUE TITLE
Incorrect output of input fields with the virtual cursor 

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#465]
### Summary

1. Go to https://www.w3schools.com/code/tryit.asp?filename=GK4VUXCLKLQB
2. read with down arrow key

### Expected result

* Role, name and value should be output once and correctly (WCAG SC 4.1.2)
* aria-invalid=true should be output correctly

### Actual result

* Role, name and value are only output correctly in IE 11. In Chrome, Edge and Firefox the role is partly not and the value is partly wrong (the previous value at the next field). In Firefox, name and value are also output three times
* aria-invalid=true is not output correctly: in IE 11 there is no output, in Chrome, Edge and Firefox there is also output for the following field

The problem occurs only in Auto Forms Mode. In SemiAuto mode the problems do not occur with one exception. Only in Firefox, even with "SemiAuto" each form label is output twice

### IE 11

First Name: 
Edit Susan
Last Name: 
Edit Smith
Email address: 
Edit test-email
Phone: 
Edit 040-23423421

### Chrome 86.0.4240.111 and Edge 86.0.622.58 

First Name: 
Susan
Last Name: 
Susan
Edit 
Smith
Email address: 
Edit 
Invalid test-email
Phone: 
end Invalid 
Edit 
invalid entry 
040-23423421

### Firefox 81.0.2 ESR

heading level 1 HTML Links
First Name: 
First Name:
First Name: Edit 
Type in text.
Susan
Susan
Last Name: 
Last Name:
Susan
Edit 
Smith
Smith
Smith
Email address: 
Email address:
Smith
Edit 
Invalid test-email
test-email
Phone: 
Phone:
test-email
Edit 
invalid entry 
invalid entry 
end Invalid 040-23423421
040-23423421
040-23423421

#### JAWS version and build number
2020.2008.24
 
#### Operating System and version
Windows 8

**Comment by maxharney** [comment]
+1 

I'm able to reproduce this issue as well on Windows 10 and Chrome in Auto forms mode with the virtual cursor. Has anyone had a chance to look into this?

**Comment by JAWS-test** [comment]
A year and a half later and nothing has happened (https://github.com/FreedomScientific/VFO-standards-support/issues/614)
